templates/test.template

```diff
@@ -2,21 +2,20 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-template: default
 author: Mathias Bynens
-desc: Unicode property escapes `\p{<%= UnicodePropertyValueExpression %>}` and `\P{<%= UnicodePropertyValueExpression %>}`
+description: >
+  Unicode property escapes `\p{}` and \P{}` for `<%= UnicodePropertyValueExpression %>`
 esid: sec-static-semantics-unicodematchproperty-p
-flags: [generated]
```

Comment:
this is already generating valid test files, this flag would be only used for the original test262 generation tool which does not really apply to these cases.

With this changes, we need to rename desc to description and not reference template anymore.